src/widgets/FormRow.js

```diff
@@ -65,7 +65,8 @@ new Adapter({
     implements: IFormRowWidget,
     adapts: interfaces.IBaseField,
     
-    Component: Row
+    Component: Row,
+    doesNotRenderLabel: false
```

Comment:
From what I can tell, this is determined by ` {field.label && ` on row 53. Which means it should be evaluated at runtime like this:

doesNotRenderLabel: function () { return this.context.label ? false : true }

Reply:
Note: This applies to all instances if doesNotRenderLabel